apps/docs/content/components/buttons/LinkButton.mdx

```diff
@@ -33,9 +33,7 @@ links:
 
 A `LinkButton` uses the following components.
 
-- [Icon](./Icon)
-- [IconList](./IconList)
-- [Text](./Text)
+<ComposedComponents components={["Icon", "IconList", "Text"]}/>
```

Comment:
It would be nice if instead of declaring the components in alpohabetical order, that the sorting would be done directly inside the composedcomponents component